src/components/MenuPanel/MenuPanel.js

```diff
@@ -159,14 +161,15 @@ class MenuPanel extends React.PureComponent {
     )
   }
   renderLoadedAppGroup = apps => {
-    const { appsLoading } = this.props
+    const { appsStatus } = this.props
 
     // Wrap the DAO apps in the loader
     return (
       <MenuPanelAppsLoader
         key="menu-apps"
-        loading={appsLoading}
+        appsStatus={appsStatus}
         itemsCount={apps.length}
+        onRetry={this.props.onRequestAppsReload}
```

Comment:
Let's destruct this too in https://github.com/aragon/aragon/pull/365/files#diff-310af66f20aed127fd841958f3014ce6R164 😊